scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -644,9 +644,6 @@ def load_from_file(self, file, filter_platform=[]):
                     if ts.get("run_id"):
                         instance.run_id = ts.get("run_id")
 
-                    if self.options.device_testing:
```

Comment:
better to remove this late unfortunate option assignment, although other side effects might also pop up...

Reply:
sorry need some time to link, e.g. at this one
https://github.com/zephyrproject-rtos/zephyr/blob/31fea97e05fd524b089c70281356d07df7788349/scripts/pylib/twister/twisterlib/testplan.py#L726